mobile/src/src/components/Housing/Listings/ListingsContainer.jsx

```diff
@@ -42,18 +42,53 @@ class ListingContainer extends Component {
   };
 
   likeListing = (id, shouldLike) => {
+    // updates number of likes
+    let index = this.state.listings.findIndex((obj) => obj.id == id);
+    let copyList = JSON.parse(JSON.stringify(this.state.listings));
+    shouldLike == true
```

Comment:
Nitpick: you don't have to do `shouldLike == true?`, you can instead write `shouldlike?`.

Reply:
Will do